R/R/metaflow_client.R

```diff
@@ -93,7 +91,7 @@ metaflow_object <- R6::R6Class(
       private$created_at_ <- obj$created_at
       private$parent_ <- obj$parent$id
       private$pathspec_ <- obj$pathspec
-      private$tags_ <- import_builtins()$list(obj$tags)
+      private$tags_ <- reticulate::import_builtins()$list(obj$tags)
```

Comment:
What's the rationale for introducing `reticulate::` here and elsewhere in lieu of NAMESPACE import?

Reply:
We don't have import reticulate in NAMESPACE anymore. Only import we have is `importFrom(magrittr,"%>%")` in NAMESPACE. This is presumably better practice since it does not expose all the methods in dependency libraries. 

`reticualte::import_builtins` is also better for readability as it's clear we're using a method from `reticulate`